consensus/istanbul/backend/snapshot_test.go

```diff
@@ -256,7 +258,8 @@ func TestValSetChange(t *testing.T) {
 			headers: make(map[uint64]*types.Header),
 		}
 
-		engine := New(config, db).(*Backend)
+		dataDir := filepath.Join("/tmp", string(rand.Int()))
```

Comment:
1. I couldn't find one in the codebase.
2. I tried importing but it didn't work, so, I decided to duplicate. Since this duplication is only in the test code, I think it is not a major issue.